exec_test.go

```diff
@@ -35,3 +36,42 @@ func ExampleCommandExists() {
 	// true
 	// false
 }
+
+func BenchmarkCommandExists(b *testing.B) {
+	cases := []struct {
+		Command string
+	}{
+		{"go"},
+		{"gofmt"},
+		{"bazel"},
+		{"ps"},
```

Comment:
what is the goal to make 4 tests on real binaries?

what the second, third, and fourth benchmarks may give as additional information?

the only interesting test I can see to add is something that will always be missing from any computer running the test; a binary that is always command-not-found

Reply:
@moul Yes, I'm missing the failed cmd dummies.